rllib/tuned_examples/mbmpo/halfcheetah-mbmpo.yaml

```diff
@@ -1,11 +1,12 @@
-halfcheetah-mb-mpo:
-    env: ray.rllib.examples.env.halfcheetah.HalfCheetahWrapper
+halfcheetah-mbmpo:
```

Comment:
It would really be nice to have a small learning test case for MBMPO in our regression suite. Could we do the following?
- wrap a very simple cont. action env? Like some repeat_after_me env (where we would be able to write a reward function).
- Add a MBMPO regression test to BUILD?

Reply:
One thing to keep in mind is that each gradient step in MBMPO does take a while (2-3 second), after all, higher order gradient is caculated in MAML. In addition, one needs to go through many epochs of training data for the TD models before even going through MAML.

Reply:
Even if it is a simple env, it might take a while... I'm wondering what's the max time allocated for regression tests?

Reply:
@sven1977

Reply:
Some take a couple of minutes (I think DDPG or PPO Pendulum are the slowest ones), which is totally fine. Would anything (very simple) be possible to learn in let's say 5min?

Reply:
Not sure XD I'll try it out on pendulum and lyk how long it takes to get past -800 rew

Reply:
Aight it does not seem possible, b/c training models to a decent level first to sample from takes a while....